### PR TITLE
feat(docs): update generators comparison

### PR DESCRIPTION
**Pull request by tknickman**
Updating the comparison table to reflect some recent Turborepo releases (generators, and run summaries).

### CODE REVIEW

File: website/src/components/ComparisonTable.tsx [hunk 1]
```diff
@@ -87,6 +87,7 @@ const workspaceRows: Comparison[] = [
 		support: {
 			moon: SUPPORTED,
 			nx: SUPPORTED,
+			turborepo: SUPPORTED,
```
Comment (tknickman) [inline website/src/components/ComparisonTable.tsx:90]: https://turbo.build/blog/turbo-1-10-0#code-generators

File: website/src/components/ComparisonTable.tsx [hunk 2]
```diff
@@ -410,14 +411,15 @@ const taskRunnerRows: Comparison[] = [
 		feature: 'Can generate run reports',
 		support: {
 			moon: SUPPORTED,
+			turborepo: SUPPORTED,
```
Comment (tknickman) [inline website/src/components/ComparisonTable.tsx:414]: https://turbo.build/blog/turbo-1-9-0#view-and-compare-task-runs

File: website/src/components/ComparisonTable.tsx [hunk 3]
```diff
@@ -410,14 +411,15 @@ const taskRunnerRows: Comparison[] = [
...
 		},
 	},
 	{
 		feature: 'Continuous integration (CI) support',
 		support: {
 			moon: SUPPORTED,
 			nx: PARTIALLY_SUPPORTED,
-			turborepo: PARTIALLY_SUPPORTED,
+			turborepo: SUPPORTED,
```
Comment (tknickman) [inline website/src/components/ComparisonTable.tsx:422]: https://turbo.build/repo/docs/ci

Comment (milesj) [reply website/src/components/ComparisonTable.tsx:422]: This isn't correct. Turbo doesn't have a built-in CI method similar to `moon ci`.

**Comment by tknickman** [comment]
Fixed @milesj, thanks for the review!

**Comment by milesj** [comment]
@tknickman Awesome thanks!

**Pull request merged**